### ISSUE TITLE
Fails to parse usage of variable `of`

### ISSUE THREAD

**Issue by privatenumber** [guybedford/es-module-lexer#173]
## Reproduction
```js
import { init, parse } from 'es-module-lexer'

await init

parse('const of=1;console.log(of/1)')
```

StackBlitz: https://stackblitz.com/edit/stackblitz-starters-ku65d9?file=index.mjs

Although `of` is an identifier with a special meaning, it is not reserved:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#identifiers_with_special_meanings

This also runs fine in Node/Chrome: `const of=1;console.log(of/1)`

**Comment by guybedford** [comment]
Thanks for posting, I've posted a fix in https://github.com/guybedford/es-module-lexer/pull/174.

**Comment by privatenumber** [comment]
Thanks for the quick turn around! Really appreciate it 🙌

BTW while I have you here, do you take sponsorships for this project? I only see JSPM and SystemJS on your Open Collective.

**Comment by guybedford** [comment]
Released in 1.5.3.

You're always welcome @privatenumber. Sponsorships to JSPM are always a huge help as it subsidizes the server costs, and is done in a tax efficient way.
